### ISSUE TITLE
pickups must not have shadow generated

### ISSUE THREAD

**Issue by BrutalSystem** [assaultcube/AC#377]
reasons to request a change:
in some cases, it's possible to tell if the pickup has resurfaced because shadows go through walls.

**Comment by flowtron** [comment]
Those shadows are rendered without any concern for geometry, which makes it fast – I'd argue this should (if at all) be addressed by mappers: have the engine in mind you're mapping for. 

- Not having shadows at all would be a serious blow to visual believability (more than seeing a shadow through a wall on a "My First Map")
- If the wall isn't any real obstruction to reaching the pickup the benefit is negligible.
- If the wall is an obstruction _that_ negates the benefit of knowing it has spawned.

**Comment by Calinou** [comment]
I also agree that pickup shadows should be left in. If the map cannot be modified to resolve this, then the shadow length should be reduced in the map. However, this isn't possible yet: https://github.com/assaultcube/AC/issues/422
